test_pytest_html.py

```diff
@@ -25,6 +25,23 @@ def run(testdir, path='report.html', *args):
     return result, html
 
 
+def assert_summary_filter(html, test_outcome='Passed', test_outcome_number=1):
```

Comment:
This is always called with an outcome and total, so I don't think they need to be optional arguments.